src/Uno.UI/UI/Xaml/Controls/DropDownButton/DropDownButton.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Automation.Peers;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class DropDownButton : Button
+	{
+		public DropDownButton()
+		{
+			DefaultStyleKey = typeof(PersonPicture);
+		}
+
+		protected override AutomationPeer OnCreateAutomationPeer()
+		{
+			return new DropDownButtonAutomationPeer(this);
+		}
+
+		//  - Missing
+		//	m_flyoutPropertyChangedRevoker = RegisterPropertyChanged(*this, winrt::Button::FlyoutProperty(), { this, &DropDownButton::OnFlyoutPropertyChanged });
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5101549)